lib/travis/build/script/shared/directory_cache/s3.rb

```diff
@@ -94,11 +93,11 @@ def push
           end
 
           def fetch_url(branch = group)
-            url('GET', prefixed(branch), expires: fetch_timeout)
+            url('GET', prefixed(branch, '.tbz'), expires: fetch_timeout)
```

Comment:
I'm assuming the intent here is to migrate to `.tgz` along the way, so won't this mismatch of downloading `.tbz`, then uploading `.tgz` only work the first time?